### PR TITLE
Removed debug messages for h5py

### PR DESCRIPTION
**Pull request by lucas-wilkins**
Disabled the Converting from 5 to 7 etc messages via log

### CODE REVIEW

File: src/sas/system/log.ini [hunk 1]
```diff
@@ -41,7 +41,7 @@ args=(os.path.join(os.path.expanduser("~"),'sasview.log'),"a")
 # Loggers
 
 [loggers]
-keys=root,saspr,sasgui,sascalc,sasmodels
+keys=root,saspr,sasgui,sascalc,sasmodels,h5py
```
Comment (rozyczko) [inline src/sas/system/log.ini:44]: h5py is not "ours", so it has no place in these keys.
3rd party loggers should probably be all set by default to at least DEBUG if not WARNING, with those that need detailed logging whitelisted.
https://github.com/SasView/sasview/pull/2275#discussion_r1005979992

**Review by rozyczko — CHANGES_REQUESTED** [review]

File: src/sas/system/log.ini (hunk 1 above)
Comment (lucas-wilkins) [reply src/sas/system/log.ini:44]: @rozyczko If you know a better way to do it, please do. I'm definitely not sure the solution I proposed is good.

Comment (lucas-wilkins) [reply src/sas/system/log.ini:44]: I asked everyone downstairs if they knew the right way of disabling these messages, and nobody did.

Comment (pkienzle) [reply src/sas/system/log.ini:44]: Changing lines 36 and 47 from DEBUG to WARNING didn't do it?

Comment (lucas-wilkins) [reply src/sas/system/log.ini]: That works, is generally better.

**Review by rozyczko — APPROVED** [review]
Less junk in the log explorer/command prompt is always good.
Easy to change the .ini settings to increase the logging level, if required.

**Pull request merged**